corehq/apps/sms/models.py

```diff
@@ -196,7 +195,7 @@ class SMSBase(UUIDGeneratorMixin, Log):
             ugettext_noop("The gateway can't reach the destination number."),
         ERROR_MESSAGE_TOO_LONG:
             ugettext_noop("The gateway could not process the message because it was too long."),
-        ERROR_MESSAGE_BLANK:
+        'MESSAGE_BLANK':
```

Comment:
What's this change for?

Reply:
I removed it from the constant since it's not in use in the code but it needs to be in this still since it may be referenced in the DB. Does that make sense?

Reply:
yep, got it